Docs/source/usage/parameters.rst

```diff
@@ -951,6 +951,17 @@ Particle initialization
 
           \sigma_{x,y}(z) &= \sigma^*_{x,y} \sqrt{1 + \left( \frac{z - z^*}{\beta^*_{x,y}} \right)^2}
 
+      * ``<species_name>.do_rotation`` (`bool`, optional) the beam is rotated around its centroid.
```

Comment:
Could we change the user-input name, to make it clear that this argument is specific to the "gaussian_beam" initialization?
```suggestion
      * ``<species_name>._rotation`` (`bool`, optional) the beam is rotated around its centroid.
```

Reply:
In the text, could you also add that this can be used e.g. to initialize beams that are crossing at an angle.